release/README.md

```diff
@@ -50,6 +52,19 @@ This script does the following:
 5. Uses `skaffold` to build and push new stable release images to `gcr.io/bank-of-anthos`.
 6. Pushes the Git tags and the release branch.
 
+### Troubleshooting script failures
+
+In the event of any of the steps above failing you might have to revert the repository to it's original state. Follow the steps below as required:
```

Comment:
s/it's/its

Reply:
Fixed in 7bb9544363ac55987a066e55c24a0fff2b5ab3b6